src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetControllerTests.java

```diff
@@ -63,6 +61,11 @@
                                                 "/company-accounts/" + COMPANY_ACCOUNTS_ID +
                                                 "/small-full/approval";
 
+    private static final String REVIEW_PATH = "/company/" + COMPANY_NUMBER +
```

Comment:
The `APPROVAL_PATH` above should be redundant now and can be removed

Reply:
Removed. 👍